### PR TITLE
Minor improvements to user_test.rb.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: dashboard/test/models/user_test.rb [hunk 1]
```diff
@@ -177,12 +177,9 @@ class UserTest < ActiveSupport::TestCase
 
     user.age = 24
     assert_equal "21+", user.age
-    assert user.age
```
Comment (ashercodeorg) [inline dashboard/test/models/user_test.rb:180]: No reason to assert truthiness after asserting its exact value.

File: dashboard/test/models/user_test.rb [hunk 2]
```diff
@@ -177,12 +177,9 @@ class UserTest < ActiveSupport::TestCase
...
 
     user.save!
-
-    user = User.find(user.id)
-
+    user.reload
```
Comment (ashercodeorg) [inline dashboard/test/models/user_test.rb:182]: No reason to refind the user when the test is checking for persistence of the value to the DB.

Comment (Hamms) [reply dashboard/test/models/user_test.rb:182]: is there a significant difference between `reload` and `find`? I don't see how this update changes much of anything

**Comment by ashercodeorg** [comment]
No, there isn't a huge difference between `reload` and `find`. But I'd say the former is clearly preferable to the latter, in this case.

**Pull request merged**